src/ui/templates/results/pagination.hbs

```diff
@@ -56,7 +56,9 @@
 
     {{#each pageNumbers}}
       {{#if this.active}}
-        <span id="active-page" class="yxt-Pagination-page" aria-label="Currently on page {{../pageNumber}}">{{../pageLabel}} {{../pageNumber}}</span>
+        <span id="active-page" class="yxt-Pagination-page {{#if this.activeAll}}yxt-Pagination-active-all{{/if}} {{#if this.onePageAll}}yxt-Pagination--onepage-all{{/if}}
+        {{#if this.activeMobile}}yxt-Pagination--active-mobile{{/if}}{{#if this.activeDesktop}}yxt-Pagination--active-desktop{{/if}}" 
```

Comment:
Can we add spaces here for when the if statements evaluate to true? Are they one class if we don't add this?

e.g. `{{#if this.activeDesktop}} yxt-Pagination--active-desktop{{/if}}`